### DISCUSSION TITLE
YouTube v. 19.47.53 - minimizing app + locking phone stops playback

### DISCUSSION THREAD

**Discussion by vsgj** [ReVanced/revanced-discussions#1130] [General] [answered]
Hi, posting here because I can't find a solution anywhere else. I have updated Revanced manager and microG Services (disabled any battery restrictions as well). 

The adblocker is working, but when you leave the app or lock the phone, the video stops. This is an issue that's appeared in December 2024. I found a Reddit user who had the same issue, but apart from that I can't see anyone else writing about this.

I will test older versions tomorrow to see if they work just in case.

Could this be a patch issue or something I am missing from my phone (fully updated Android)?

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Repatch using default patch selection.

**Reply by vsgj** [reply]
Thank you, that worked! I was worried about messing with the default settings, but it wasn't hard to find luckily

**Comment by Amanbhatti02** [comment]
I have same issue, i have repatched by default patch selection, also i have removed any restriction from battery usage . But still video stops on minimising and locking the screen. @LisoUseInAIKyrios
